client/keys/utils.go

```diff
@@ -71,3 +72,17 @@ func printTextRecords(w io.Writer, kos []KeyOutput) error {
 
 	return nil
 }
+
+// displayDiscreetly Print a secret string to an alternate screen, so the string isn't printed to the terminal.
+func displayDiscreetly(clientCtx client.Context, w io.Writer, promptMsg, secretMsg string) (err error) {
+	output := termenv.NewOutput(w)
+	output.AltScreen()
+	defer output.ExitAltScreen()
+	if _, err = fmt.Fprintf(output, "%s\n\n%s\n\nPress 'Enter' key to continue.", promptMsg, secretMsg); err != nil {
+		return err
+	}
+	if _, err = fmt.Scanln(); err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit: since we're wrapping the error with context (maybe we should), then you can just simply return the result

```suggestion
	return fmt.Scanln()
```

Reply:
@alexanderbez Hi, sir. As `fmt.Scanln` signature is `func Scanln(a ...any) (n int, err error)`, but I just wanna return "error or not" for `func displayDiscreetly`